src/GUI/EFCorePowerTools/EFCorePowerTools.csproj

```diff
@@ -170,6 +170,7 @@
     <Content Include="efpt\2.1.3\efpt.exe.config">
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
+    <None Include="efpt.renaming.json" />
```

Comment:
Did you want to provide this just as a sample?

Reply:
Yes. Just only as sample but didn't know where to put it.

Reply:
Yes, I need to think of a good way to make this feature not too hidden.

Let's get this in and think of a good way to do that later.